### ISSUE TITLE
chd conversion script only erases *.cue and *.bin even when its handling *.iso or *.gdi

### ISSUE THREAD

**Issue by joyrider3774** [dragoonDorise/EmuDeck#103]
Check the RM parts at the end of these lines. It only erases *.cue and *.bin while it could be handling the gdi or iso file
https://github.com/dragoonDorise/EmuDeck/blob/main/tools/chdconv/chddeck.sh#L38

**Comment by joyrider3774** [comment]
it should be changed to this i think

```
find "$romsPath" -not -path "$romsPath/psp/*" -type f -name "*.cue" | while read f; do chdman5 createcd -i "$f" -o "${f%.*}.chd" && rm -rf "${f%.*}.cue" && rm -rf "${f%.*}.bin"; done;
	find "$romsPath" -not -path "$romsPath/psp/*" -type f -name "*.gdi" | while read f; do chdman5 createcd -i "$f" -o "${f%.*}.chd" && rm -rf "${f%.*}.gdi"; done;
	find "$romsPath" -not -path "$romsPath/psp/*" -type f -name "*.iso" | while read f; do chdman5 createcd -i "$f" -o "${f%.*}.chd" && rm -rf "${f%.*}.iso"; done;
```

**Comment by Godsbane** [comment]
the more i think about this, the more i think once we have created the chd, we should delete all things that share the same root name except the chd itself... rather than trying to assume we know all the different possible file types.

**Comment by neoKushan** [comment]
I don't know if this warrants a new issue, but it seems the conversion script isn't deleting the .bin portion of a bin/cue file. I'm just converting a load of PS1 and PS2 titles now and it seems to consistently be leaving a .bin next to a .chd file. 

This is a brand new install of EmuDeck on a brand new Steam Deck (Only arrived today).

**Comment by Godsbane** [comment]
Thanks, I'll check it out. Probably the regex.

**Comment by neoKushan** [comment]
Let me know if I can help at all, happy to feed back any info that'll help diagnose it :)

**Comment by Godsbane** [comment]
This is probably ready for testing.

**Comment by nderscore** [comment]
I just ran into a similar issue using the CHD script.

It failed to clean up all of the `.bin` files of a multi-track image (a Playstation game where each CD-audio track was ripped as a separate `.bin`)

When archiving a `.cue` file, I don't think it's safe to assume that the filename of the `.bin` file(s) will be the same. In theory, a completely different filename(s) could be referenced in the cue sheet.

**Comment by Godsbane** [comment]
Did you test the dev/beta version? This issue should be fixed there. Let me know.

**Comment by nderscore** [comment]
> Did you test the dev/beta version? This issue should be fixed there. Let me know.

@Godsbane that's my bad. I was looking at the version of the CHD script that's on `main`. The version on `dev` looks like it should address my concerns :+1: (though I'll admit I haven't tested it yet :zipper_mouth_face: )

**Comment by dragoonDorise** [comment]
Fixed in 0.17.6
